onnxruntime/test/providers/cpu/math/matmul_integer_test.cc

```diff
@@ -30,5 +30,54 @@ TEST(MatmulIntegerOpTest, MatMulInteger) {
   test.AddOutput<int32_t>("T3", {1, 1}, {-1});
   test.Run();
 }
+
+TEST(MatmulIntegerOpTest, MatMulInteger_Uint8_Int8_GEMM) {
```

Comment:
None of the test cases which have int8 inputs actually have any -ve numbers... it would be good to have some negative numbers too... It will help to test this better

Reply:
Will use randomly generated data.

---
In reply to: [303687073](https://github.com/microsoft/onnxruntime/pull/1391#discussion_r303687073) [](ancestors = 303687073)